Undefined variable `backend` in statedump directory name

Running a make without using EXTRA_VARS to setup a backend appears to be failing because the variable backend is not set.

```
[sprabhu@splp2 sit-environment (main)]$ make
make[1]: Entering directory '/data/sprabhu/src/ocs/sit-environment/playbooks'

PLAY [localhost] *****************************************************************************************************************************************************************************************************************************

TASK [Gathering Facts] ***********************************************************************************************************************************************************************************************************************
Wednesday 29 November 2023  22:26:01 +0000 (0:00:00.006)       0:00:00.006 **** 
ok: [localhost]

TASK [local.defaults : Load settings] ********************************************************************************************************************************************************************************************************
Wednesday 29 November 2023  22:26:02 +0000 (0:00:01.064)       0:00:01.070 **** 
ok: [localhost]

TASK [local.defaults : Generate config.yml in ansible directory] *****************************************************************************************************************************************************************************
Wednesday 29 November 2023  22:26:02 +0000 (0:00:00.013)       0:00:01.083 **** 
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ansible.errors.AnsibleUndefinedVariable: 'backend' is undefined. 'backend' is undefined
fatal: [localhost]: FAILED! => {"changed": false, "msg": "AnsibleUndefinedVariable: 'backend' is undefined. 'backend' is undefined"}

NO MORE HOSTS LEFT ***************************************************************************************************************************************************************************************************************************

PLAY RECAP ***********************************************************************************************************************************************************************************************************************************
localhost                  : ok=2    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   

Wednesday 29 November 2023  22:26:02 +0000 (0:00:00.077)       0:00:01.161 **** 
=============================================================================== 
Gathering Facts ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 1.06s
local.defaults : Generate config.yml in ansible directory ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 0.08s
local.defaults : Load settings -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 0.01s
make[1]: *** [Makefile:10: local] Error 2
make[1]: Leaving directory '/data/sprabhu/src/ocs/sit-environment/playbooks'
make: *** [Makefile:4: test] Error 2
```

We should default to xfs if no backend is defined.

--- a/playbooks/roles/local.defaults/templates/config.yml.j2
+++ b/playbooks/roles/local.defaults/templates/config.yml.j2
@@ -99,5 +99,5 @@ config:
     {%- endfor +%}
   {%- endfor +%}
 
-  statedir: "{{ misc.host.statedir }}/sit.{{ backend }}_statedump"
+  statedir: "{{ misc.host.statedir }}/sit_statedump"
   configdir: "{{ misc.host.configdir }}"

